src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/reroute/actions/PostResourceAllocationAction.java

```diff
@@ -62,10 +62,12 @@ protected Optional<Message> performWithResponse(State from, State to, Event even
             currentForwardId = flow.getForwardPathId();
         }
         FlowPath currentForwardPath = currentForwardId != null ? getFlowPath(currentForwardId) : null;
-
+        Message rerouteResponse = buildRerouteResponseMessage(currentForwardPath, newForwardPath,
+                stateMachine.getCommandContext());
         if (stateMachine.getNewPrimaryForwardPath() == null && stateMachine.getNewPrimaryReversePath() == null
                 && stateMachine.getNewProtectedForwardPath() == null
                 && stateMachine.getNewProtectedReversePath() == null) {
+            stateMachine.setOperationResultMessage(rerouteResponse);
```

Comment:
What will happen if we would find new path, because it doesn't exists. I mean current path is broken and there is no other path. Will we return "Flow already on the best path"?

Reply:
The following message:

{
  "correlation_id": "57501d77-f954-4771-8cd1-b5e126271cc7 : sdasd",
  "timestamp": 1690970708697,
  "error-type": "Object was not found",
  "error-message": "Could not reroute y-flow",
  "error-description": "Not enough bandwidth or no path found. Switch 00:00:00:00:00:00:00:08 doesn't have links with enough bandwidth, Failed to find path with requested bandwidth=100"
}